ros/api/v0/hosts.py

```diff
@@ -13,12 +13,35 @@
     offset_value)
 import logging
 
+
 LOG = logging.getLogger(__name__)
 SYSTEM_STATES_EXCEPT_OPTIMIZED = [
     "Oversized", "Undersized", "Idling", "Storage rightsizing"
 ]
 
 
+class IsROSConfiguredApi(Resource):
+    def get(self):
+        ident = identity(request)['identity']
+        account_query = db.session.query(RhAccount.id).filter(
+            RhAccount.account == ident['account_number']).subquery()
+        system_count = db.session.query(System.id)\
+            .filter(System.account_id.in_(account_query)).count()
+
+        if system_count <= 0:
+            return {
+                'success': False,
+                'code': 'NOSYSTEMS',
+                'count': system_count
+            }
+
+        return {
+            'success': True,
+            'code': 'SYSTEMSEXIST',
+            'count': system_count
+        }
```

Comment:
Can we instead use an if-else block with a single return statement? Either way works.

```suggestion
        if system_count <= 0:
            success = False
            systems_code = 'NOSYSTEMS'
        else:
            success = True
            systems_code = 'SYSTEMSEXIST'

        return {
            'success': success,
            'code': systems_code,
            'count': system_count
        }       
```

Reply:
@apuntamb,
Thank you for review. 
Here, I would prefer to return directly `dict` objects instead of declaring an extra variables.  
In case where it requires multiple keys & operations to perform based on condition & on this dict object before returning final result then definitely I would have preferred `if-else` loop.
Any thoughts? otherwise I will go ahead and modify the change as per your suggestion.

Reply:
Yeah sure, makes sense! We can go ahead with the current implementation.